src/components/draft-js-editor/DraftJSEditor.js

```diff
@@ -117,13 +121,17 @@ class DraftJSEditor extends React.Component<Props> {
                     {label}
                     {!isRequired && <OptionalFormattedMessage />}
                 </span>
+                <span className={classNames({ 'accessibility-hidden': true })} id={this.commentFormTip}>
```

Comment:
Can we simplify this to `<span className="accessibility-hidden" id={this.commentFormTip}>`?